docs/_sources/chapters/css/structure.rst.txt

```diff
@@ -0,0 +1,116 @@
+CSS Structure
+=============
+
+Writing CSS
+-----------
+
+.. index:: ! rule, ! selector
+
+Programmers can change a lot of different styling using CSS **rules**.
+A rule includes the selector and a declaration block.
+A **selector** determines which elements will be affected by the rule.
+Inside the declaration block, programmers set CSS properties to specific values.
+CSS has a lot of different properties and it would be impossible to memorize them all.
+
+.. sourcecode:: css
+
+   selector {
+       declaration block
+   }
+
+CSS has three different selectors that the programmer can use to make their style choices.
+The first one that most beginners start with is the element selector.
```

Comment:
How about giving each selector it's on section and providing an example in that section?